buildSrc/src/main/java/com/uber/okbuck/core/manager/GroovyManager.java

```diff
@@ -16,6 +16,8 @@
 import org.gradle.api.Project;
 import org.gradle.api.artifacts.Configuration;
 
+import javax.annotation.Nullable;
```

Comment:
Might want to standardize one. I see both this and jetbrains' in this PR

Reply:
catch. Will fix everywhere